server/views/admin/users/index.njk

```diff
@@ -4,69 +4,113 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/input/macro.njk" import govukInput %}
 {% from "govuk/components/pagination/macro.njk" import govukPagination %}
-
-{% set pageTitle = applicationName + " - " + pageHeading  %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% set pageTitle = applicationName + " - " + pageHeading %}
 {% set mainClasses = "app-container govuk-body assessments--index" %}
-
-{% block beforeContent %}
-    {{ govukBackLink({
-		text: "Back",
-		href: '/'
-	}) }}
-{% endblock %}
-
+{% block beforeContent %}{{ govukBackLink({"text": "Back", "href": "/"}) }}{% endblock %}
 {% block content %}
-
     {% include "../../_messages.njk" %}
     <div class="govuk-grid-row">
-
-        <h1 class="govuk-heading-l govuk-grid-column-three-quarters">{{ pageHeading }}</h1>
-
-        {{ govukButton({
-                                text: "Add new user",
-                                href: paths.admin.userManagement.new({}),
-                                classes: "govuk-grid-column-one-quarter"
-                }) }}
-
-        <div class="grid-column-full">
-            <form action="{{ paths.admin.userManagement.search({}) }}" method="post">
-                <input type="hidden" name="_csrf" value="{{ csrfToken }}">
-
+        <div class="govuk-grid-row">
+            <h1 class="govuk-heading-l govuk-grid-column-three-quarters">{{ pageHeading }}</h1>
+            {{ govukButton({
+                        text: "Add new user",
+                        href: paths.admin.userManagement.new({}) ,
+            classes: "govuk-grid-column-one-quarter"
+            }) }}
+        </div>
+        <form action="{{ paths.admin.userManagement.search({}) }}" method="post">
+            <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+            <div class="govuk-grid-row">
```

Comment:
`div`s with `govuk-grid-row` classes shouldn't be nested as it will cause layout issues. [The documentation](https://design-system.service.gov.uk/styles/layout/) has some examples that may help with using the layout classes.